scheduler/src/cook/mesos/sandbox.clj

```diff
@@ -129,6 +139,7 @@
   [{:keys [pending-sync-hosts] :as pending-sync-state}]
   (counters/clear! sandbox-pending-sync-host-count)
   (when (seq pending-sync-hosts)
+    (prom/set prom/mesos-pending-sync-host-count (count pending-sync-hosts))
```

Comment:
Should this be inc?

Reply:
I was missing the call to set the counter value to 0 (like we do for the codahale counter with the call to clear!). Updated to have the call to set to 0, and then it's functionally equivalent to set or inc by that amount.